src/App.css

```diff
@@ -1,7 +1,76 @@
-h2.range {
-  text-align: center;
-  margin-top: 20px;
-  margin-bottom: 20px;
+:root {
+  --white-two: #fff;
+  --azure: #0290ff;
+}
+body {
+  width: 100rem;
```

Comment:
- [ ] I need to scroll from `left to right` to see the whole content. That means the whole content is not `fitting` within my browser interface. That shouldn't be the issue as per the given design. Kindly fix this issue.

![image](https://github.com/francksefu/booksmr/assets/60083437/bbf62527-f753-41b3-a217-c6607334acd3)